royal/ext/sqla.py

```diff
@@ -0,0 +1,168 @@
+import logging
+
+from pyramid.location import lineage
+import royal
+from sqlalchemy.orm.collections import MappedCollection
+
+log = logging.getLogger(__name__)
+
+
+def includeme(config):
+    config.scan(__name__)
+
+
+class Collection(royal.Collection):
+    sa_model = None
+    sa_exceptions = None
+    entity_cls = None
```

Comment:
This ^ doesn't look decent.. But I've tried it with zope interfaces and it doesn't look any prettier too.. Suggestions ?